Alter primary key (#1487)
Codecademy/docs: content/sql/concepts/primary-keys/primary-keys.md

);
```

> If you have created a table that doesn't have a primary key, you can alter it to add a one.

Reviewer: ```suggestion
If a table doesn't have a primary key, it can be altered to add one.
```
